### ISSUE TITLE
Header: Active navigation item in HCM not indicated clearly

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#2247]
The active menu item should be clearly indicated in the internet header.

Megadropdown closed:
![image](https://github.com/swisspost/design-system/assets/1659006/71e6a691-4c8b-4f7c-99a4-6ca09670863b)

Megadropdown open:
![image](https://github.com/swisspost/design-system/assets/1659006/4a63fd59-b928-4a9e-9079-49393c2c3c04)

Example: Google Chrome active tab
![image](https://github.com/swisspost/design-system/assets/1659006/ef6f6b65-74ad-497c-9633-ffe85f7b8534)
